tempesta_fw/t/unit/test_http_parser.c

```diff
@@ -1601,10 +1601,6 @@ TEST(http_parser, accept)
 		"Accept: text/html; q = 0.5 \r\n"
 		"\r\n");
 
-	EXPECT_BLOCK_REQ("GET / HTTP/1.1\r\n"
-		"Accept: */invalid\r\n"
-		"\r\n");
```

Comment:
This was a requirement from https://github.com/tempesta-tech/tempesta/issues/1203 : `Accept` header is security sensitive and can be used to bypass WAFs. Frankly, to fix current issue it's enough to not to set `TFW_HTTP_B_ACCEPT_HTML` for `*/*` value. But we still do need to accurately process the header.

Reply:
Yeah, missed that. Degradation is fixed and parser function was made a bit strict.